massgov/mayflower
# Patternlab/DP-15521  campaign full width component

## Description
Build Campaign Full Width Header component v 0.1 and Marketing Campaign Page v 0.1

## Related Issue / Ticket

- [DP-15521](https://jira.mass.gov/browse/DP-15521)

## Steps to Test

1. Go to [The campaign header full width component](https://mayflower.digital.mass.gov/b/patternlab/DP-15521--Campaign-full-width-component/patterns/03-organisms-by-template-campaign-full-width-header/03-organisms-by-template-campaign-full-width-header.html) 
Note that:
Because this component is the top component, the title is an H1
The background image changes based on screen size, for use with the focal point module. 

2. Go to [The campaign header full width component -- not top](https://mayflower.digital.mass.gov/b/patternlab/DP-15521--Campaign-full-width-component/patterns/03-organisms-by-template-campaign-full-width-header-not-top/03-organisms-by-template-campaign-full-width-header-not-top.html)
Note that: 
Because this component is not set as the top, the title is an H2
The background color is set to bay-blue by a section class
The text color is set to white by a section class

3. The [Marketing Campaign template](https://mayflower.digital.mass.gov/b/patternlab/DP-15521--Campaign-full-width-component/patterns/04-templates-01-content-types-marketing-campaign/04-templates-01-content-types-marketing-campaign.html) is here

4. the [Marketing Campaign Page](https://mayflower.digital.mass.gov/b/patternlab/DP-15521--Campaign-full-width-component/patterns/05-pages-marketing-campaign/05-pages-marketing-campaign.html) is here

## Screenshots
![Screen Shot 2019-08-21 at 11 56 14 AM](https://user-images.githubusercontent.com/18662734/63451903-bceee280-c40a-11e9-93a9-037fd5ecdc0f.png)
![Screen Shot 2019-08-21 at 11 55 48 AM](https://user-images.githubusercontent.com/18662734/63451947-cb3cfe80-c40a-11e9-950d-6c46da8385cd.png)
![Screen Shot 2019-08-21 at 11 58 28 AM](https://user-images.githubusercontent.com/18662734/63452065-04756e80-c40b-11e9-89a6-7e5695c7e367.png)
